.storybook/themes/news.scss

```diff
@@ -1788,6 +1788,49 @@
 				quad-chain-child-item-empty: (
 					display: contents,
 				),
+				result-list: (
+					padding-bottom: var(--global-spacing-4),
+					border-bottom: 1px solid #dadada,
+					components: (
+						attribution: (
+							display: block,
+							margin-top: var(--global-spacing-4),
```

Comment:
We should avoid margin for spacing out layout, this margin value conflicts with the heading value and cancels it's self out